store/piecestore/storage/oss.go

```diff
@@ -0,0 +1,295 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aliyun/aliyun-oss-go-sdk/oss"
+	"github.com/aliyun/credentials-go/credentials"
+	"github.com/bnb-chain/greenfield-storage-provider/pkg/log"
+)
+
+type ossStore struct {
+	client *oss.Client
+	bucket *oss.Bucket
+}
+
+func (o *ossStore) String() string {
+	return fmt.Sprintf("oss://%s/", o.bucket.BucketName)
+}
+
+func (o *ossStore) CreateBucket(ctx context.Context) error {
+	err := o.bucket.Client.CreateBucket(o.bucket.BucketName)
+	if err != nil && isErrExists(err) {
+		err = nil
+	}
+	return err
```

Comment:
Don't need to return bucket besides error?

Reply:
If users create bucket successfully, they can do some operations based on bucketName.